pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "pyparsing"
+authors = [{name = "Erik Rose", email = "erikrose@grinchcentral.com"}]
```

Comment:
Copy/pasta fail, I assume - please fix.

Reply:
Such a shameful fail, I'm sorry about that.